src/build/makers/ios.js

```diff
@@ -15,63 +15,44 @@ module.exports = function(output,lib_location,test_dir, sha, devices, entry_poin
     function log(msg) {
         console.log('[IOS] ' + msg + ' (sha: ' + sha + ')');
     }
-    if (keychain_location.length === 0 || keychain_password.length === 0) {
-        log('No keychain information. Fill that out in config.json if you want to build for iOS.');
+    log('starting iOS prepare');
+    try {
+        var projectWww = path.join(output, 'www');
+        // add the medic configuration (sha,host) to destination folder
+        var medic_config='{"sha":"'+sha+'","couchdb":"'+couchdb_cfg.host+'","couchdbext":"'+couchdb_cfg.exthost+'"}';
+        fs.writeFileSync(path.join(output, 'www','autotest','pages', 'medic.json'),medic_config,'utf-8');
```

Comment:
So, I ran this on iOS, and that file actually never comes into existence. It seems like the CouchDB URI gets passed elsewhere.